sqlalchemy_mixins/tests/test_smartquery.py

```diff
@@ -785,7 +786,7 @@ def test_explicitly_set_in_schema_subqueryload(self):
         self.assertEqual(self.query_count, 2)
 
     # TODO: implement below logic
-    @unittest.expectedFailure
+    @nose.SkipTest
```

Comment:
Nose has a decorator for expecting failure `@raises(Exception)`. See the link below:
https://nose.readthedocs.io/en/latest/testing_tools.html#nose.tools.raises